scripts/imgtool/main.py

```diff
@@ -105,6 +107,27 @@ def validate_header_size(ctx, param, value):
             "Minimum value for -H/--header-size is {}".format(min_hdr_size))
     return value
 
+def get_dependencies(ctx, param, value):
+    if value is None:
+        return None
+    else:
```

Comment:
could also just be `if value:` and the block below (no return means `return None`)

Reply:
or `if value is not None:` in case it could be zero...